docs/tutorial/agent-on-docker.md

```diff
@@ -9,7 +9,7 @@ Fashion-mnist is used as an example dataset, same as [Get Started](../tutorial/g
 ## Requirements
 
 - docker: v19.0 ~
-- go: v1.14 ~
+- go: v1.15 ~
```

Comment:
1.16 is better

Reply:
Maybe the client codes doesn't depend on Go v1.16. So I think it is okay to leave it as is.